File: loiane-training-csharp/procedural-poo/c/Ex14.cs
using System;

public class Class1
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite a 1 nota do aluno");
        double nota1 = double.Parse(Console.ReadLine());

        Console.WriteLine("Digite a 2 nota do aluno");
        double nota2 = double.Parse(Console.ReadLine());

        double media = (nota1 + nota2) / 2;

        Console.WriteLine("Nota 1:" + nota1 + "Nota 2:" + nota2);
        Console.WriteLine("Media:" + media);
        if (media < 4 && media >= 0)
        {
            Console.Write("Conceito E");
            Console.WriteLine("Reprovado")
        }
        else if (media < 6)
        {
            Console.Write("Conceito D");
            Console.WriteLine("Reprovado")
        }
        else if (media < 7.5)
        {
            Console.Write("Conceito C");
            Console.WriteLine("Aprovado")
        }
        else if (media < 9)
        {
            Console.Write("Conceito B");
            Console.WriteLine("Aprovado")
        }
        else if (media <= 10)
        {
            Console.Write("Conceito A");
            Console.WriteLine("Aprovado")
        }
        else
        {
            Console.Write("Entrada de valores incorretos");
        }
    }
}

File: loiane-training-java/basico/Exer Aula 14_15/Ex14.java
import java.util.Scanner;

class Ex14 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite 2 notas do aluno para calcular a nota semestral.");
		double note1 = scan.nextDouble();
		double note2 = scan.nextDouble();
		double average = (note1 + note2) / 2;
		String concept = "";
		String result = "";

		if (average > 0 && average < 4) {
			concept = "E";
		} else if (average >= 4 && average < 6) {
			concept = "D";
		} else if (average >= 6 && average < 7.5) {
			concept = "C";
		} else if (average >= 7.5 && average < 9) {
			concept = "B";
		} else if (average >= 9 && average < 10) {
			concept = "A";
		} else {
			System.out.println("Digite uma nota valida!");
			scan.close();
			return;
		}

		switch (concept) {
			case "A":
			case "B":
			case "C":
				result = "Aprovado";
				break;
			case "D":
			case "E":
				result = "Reprovado";
				break;
		}

		System.out.println("Notas: " + note1 + " | " + note2);
		System.out.println("Media: " + average);
		System.out.println("Conceito: " + concept);
		System.out.println("Resultado: " + result);

		scan.close();
	}
}